src/vs/workbench/parts/debug/browser/debugActions.ts

```diff
@@ -210,6 +211,28 @@ export class RunAction extends StartAction {
 	}
 }
 
+export class LaunchAction extends RunAction {
+	private quickOpenService: IQuickOpenService;
+	static ID = 'workbench.action.debug.launch';
+	static LABEL = nls.localize('launchDebugConfiguration', "Select and Start Debugging");
+
+	constructor(id: string, label: string,
+		@IDebugService debugService: IDebugService,
+		@IKeybindingService keybindingService: IKeybindingService,
+		@ICommandService commandService: ICommandService,
+		@IWorkspaceContextService contextService: IWorkspaceContextService,
+		@IFileService fileService: IFileService,
+		@IQuickOpenService quickOpenService: IQuickOpenService
+	) {
+		super(id, label, debugService, keybindingService, commandService, contextService, fileService);
+		this.quickOpenService = quickOpenService;
+	}
+
+	public run(): TPromise<any> {
+		return TPromise.as(this.quickOpenService.show('debug '));
```

Comment:
Add an isEnabled() such that this action is only enabled while you are not debugging